package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@deriv/deriv-charts",
-    "version": "1.3.2",
-    "lockfileVersion": 3,
+    "version": "1.3.1",
+    "lockfileVersion": 2,
```

Comment:
why are we changing this here?

Reply:
fixed now , it got changed during merge conflict fixes , Thanks